_integration-tests/tests/go-redis.v7/validation.json

```diff
@@ -24,8 +24,7 @@
             "span.kind": "client",
             "db.system": "redis",
             "redis.raw_command": "get test_key: ",
```

Comment:
Still used?

Reply:
This is making sure the correct tags are set on the trace...